cypress/integration/common/deal-wizard.ts

```diff
@@ -0,0 +1,91 @@
+import { Then, When } from "@badeball/cypress-cucumber-preprocessor/methods";
+
+Then("I am presented the option to choose a partner", () => {
+  cy.url().should("match", /(initiate\/token-swap$)/)
+})
+
+When("I go to previous step", () => {
+  cy.get("[data-test='wizard-previous-button']").click();
+})
+
+When("I try to proceed to next step", () => {
+  cy.get("[data-test='wizard-proceed-button']").click();
+})
```

Comment:
The same as previous with url verifying

Reply:
This one is intentionally just about clicking the button, that is trying to proceed. Because we don't want to proceed if the inputs are not valid. So this steep is used in combination with other steps that check what happened